common/common-rest/src/main/java/io/servicecomb/common/rest/AbstractRestInvocation.java

```diff
@@ -62,16 +62,12 @@ public void setHttpServerFilters(List<HttpServerFilter> httpServerFilters) {
     this.httpServerFilters = httpServerFilters;
   }
 
-  protected boolean initProduceProcessor() {
```

Comment:
pull request message must include JIRA issue id for reference, e.g JAV-416

Reply:
done